starboard/BUILD.gn

```diff
@@ -297,5 +292,25 @@ if (current_toolchain == starboard_toolchain) {
     }
   }
 }
-# TODO: Create a starboard_unittest target here and fold any gtest_target_type
-# in it, e.g. common:common_test.
+
+target( "executable", "starboard_unittests") {
```

Comment:
Note how I had to add here "executable" (ISO `(gtest_target_type`) otherwise it created a .so file. I _think_ there should be some GN magic to avoid that but didn't have time to find out :P 

@andrewsavage1 @niranjanyardi

Reply:
Guys PTAL.

Reply:
Use the `test` target template from `//testing/test.gni`

Reply:
Done.